Sources/Coin.swift

```diff
@@ -27,6 +27,8 @@ public struct Coin: Equatable {
             blockchain = .ropsten
         case Coin.ethereumClassic.coinType:
             blockchain = .ethereumClassic
+        case Coin.wanchain.coinType:
+            blockchain = .wanchain
```

Comment:
Shouldn't blockchain be Ethereum?

Reply:
Wanchain has extended transaction format, it includes transaction type. So when we sign we need to distinguish this.

This is types: https://github.com/wanchain/go-wanchain/blob/8bbb49061b4bb371ce034b72e52542beb7d67b95/core/types/transaction.go#L524

Reply:
ok